tap_egencia/streams.py

```diff
@@ -92,12 +128,11 @@ def get_records(self, *args, **kwargs) -> Iterable[Dict[str, Any]]:
                 ),
             )
 
-
             get_transaction_response = self._request(get_transaction_request, None)
 
             match get_transaction_response.status_code:
                 case 200:
-                    return super().parse_response(get_transaction_response)
+                    return iter(json.loads(get_transaction_response.content.decode())["transactions"])
```

Comment:
Do we not need to call parse_response at some point?

Reply:
The important thing is that `get_records` just needs to fetch and return the records as an iterable object.

Reply:
I actually ended up doing what I said and put get_records in the parent class since the logic is the same between the two streams, then did an override of parse_response in each stream class to return a different schema